packages/schema/exported-schema.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "10.1.0",
+  "version": "10.2.0",
```

Comment:
this will be a minor bump, but we do it separately from the PR, so you can revert this line

Reply:
10-4